src/widgets/settingspages/KeyboardSettingsPage.cpp

```diff
@@ -26,6 +26,8 @@ KeyboardSettingsPage::KeyboardSettingsPage()
     form->addRow(new QLabel("Ctrl + Shift + T"), new QLabel("Create new tab"));
     form->addRow(new QLabel("Ctrl + Shift + W"),
                  new QLabel("Close current tab"));
+    form->addRow(new QLabel("Ctrl + H"),
```

Comment:
Don't like that this is a shortcut, maybe add it when custom shortcuts are a thing

Reply:
it is a shortcut because you might want to see someone spamming - e.g. when you want to moderate spam in one channel, but not want to see spam when youre watching someone else

Reply:
from my use it is extremely useful